GimbalTest/Gimbal_Grabber_Test/Gimbal_Grabber_Test.ino

```diff
@@ -58,8 +57,8 @@ void loop() {
 
 }
 
-const int SERVO_MIN_MS = 900;
-const int SERVO_MAX_MS = 2000;
+const int SERVO_MIN_MS = 400;
```

Comment:
Was this required because the servos being used are different from last time?

Reply:
I have no idea about last time, I tested it on the servos today and it didn't have the full range of motion with those min and max values.